10/10/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Xml;
+
+namespace Mass
+{
+    class Program
+    {
+
+        static void Main()
+        {
+            Console.WriteLine("Введите целочисленные первый член последовательности и ее разность");
+            int firstNum = int.Parse(Console.ReadLine());
+            int step = int.Parse(Console.ReadLine());
+            int[] nums = new int[10];
+            nums[0] = firstNum;
+            for (int i = 1; i < 10; i++)
+                nums[i] = nums[i - 1] + step;
+            var pr = new Program();
+            pr.OutputMass(nums);
+            Console.WriteLine("Введите целое число, на которое увеличится каждый элемент массива");
+            int plus = int.Parse(Console.ReadLine());
+            pr.Summ(nums, plus);
+            pr.OddNums(nums);
+            Console.WriteLine("Введите степень, на которую увеличить каждый элемент массива");
```

Comment:
Более корректная формулировка
```suggestion
            Console.WriteLine("Введите степень, в которую нужно возвести каждый элемент массива");
```